lib/constable_web/templates/comment/edit.html.eex

```diff
@@ -1,5 +1,8 @@
 <div class="container">
   <div class="comment-edit">
+    <button class="tbds-button comment-preview-text-button">Write</button>
+    <button class="tbds-button comment-preview-markdown-button">Preview</button>
```

Comment:
It seems we're rendering these buttons both on the `announcement/show.html.eex` and the `announcements/edit.html.eex` templates. Is this something that could go in the `_form.html` template since they both use that and since this functionality is related to that form?

Reply:
This is a wonderful suggestion. I'll refactor the buttons so that they live inside `comment/_form.html.eex`.

I've noticed some logic around submitting the form in `comment-form.js` and submitting, so I'll explicitly set these buttons to be of type "button" since most browsers set the default type of button as "submit".